package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "start": "./node_modules/.bin/ng serve",
     "build": "./node_modules/.bin/ng build --aot --prod",
-    "test": "./node_modules/.bin/ng test",
```

Comment:
`./node_modules/.bin/ng` is equivalent to `ng` when it is used in scripts section of package.json